services/workflows-service/src/document-file/document-file.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from '@nestjs/common';
+import { DocumentFileService } from './document-file.service';
+import { DocumentFileRepository } from './document-file.repository';
+import { PrismaModule } from '@/prisma/prisma.module';
+
+@Module({
```

Comment:
Does DocumentFile need to have a module or live inside a document module?

Reply:
It is a separate repository and service. Creating a controller feels like overkill but I'd like to avoid what happened with workflow service for runtime data + workflow definition as much as I can.